interface/billing/sl_receipts_report.php

```diff
@@ -512,14 +547,14 @@ function sel_diagnosis() {
 
  <tr bgcolor="#ddddff">
   <td class="detail" colspan="<?php echo ($form_proc_codefull ? 4 : 2) + ($form_procedures ? 2 : 0); ?>">
-   <?php echo xl('Totals for ') . $docname ?>
+   <?php echo xlt('Totals for ') . text($docname) ?>
   </td>
   <td align="right">
-   <?php bucks($doctotal1) ?>
+   <?php text(bucks($doctotal1)) ?>
```

Comment:
Remove the text() from all the bucks function calls. This is because it does nothing since the value is already echoed in the bucks function. So should instead should just change it in the function.